src/app/Scenes/SavedSearchAlertsList/Components/SavedSearchesList.tests.tsx

```diff
@@ -110,4 +111,35 @@ describe("SavedSearches", () => {
       expect(operation.fragment.variables.sort).toBe("NAME_ASC")
     })
   })
+
+  describe("tapping on an alert", () => {
+    it("opens the bottom sheet", () => {
+      __globalStoreTestUtils__?.injectFeatureFlags({ AREnableAlertBottomSheet: true })
+
+      renderWithRelay({
+        Me: () => ({
+          alertsConnection: {
+            edges: [
+              {
+                node: {
+                  internalID: "banksy",
+                  title: "Banksy",
+                  artworksConnection: {
+                    counts: {
+                      total: 57,
+                    },
+                  },
+                },
+              },
+            ],
+          },
+        }),
+      })
+
+      fireEvent.press(screen.getByText("Banksy"))
+      expect(screen.getByText("Edit Alert")).toBeTruthy()
```

Comment:
Personally, I find it easier to read if blocks are separated by an empty line, in this case, the test setup and the assertions.

```suggestion
      fireEvent.press(screen.getByText("Banksy"))

      expect(screen.getByText("Edit Alert")).toBeTruthy()
```

Reply:
thanks! I will change in a PR that is coming next